Formula/m/mumps.rb

```diff
@@ -0,0 +1,72 @@
+class Mumps < Formula
+  desc "MUltifrontal Massively Parallel sparse direct Solver"
+  homepage "https://mumps-solver.org/"
+  url "https://mumps-solver.org/MUMPS_5.6.2.tar.gz"
+  sha256 "13a2c1aff2bd1aa92fe84b7b35d88f43434019963ca09ef7e8c90821a8f1d59a"
+  license "CECILL-C"
+
+  # Core dependencies
+  depends_on "gcc"
+  depends_on "openblas"
+
+  def install
+    make_args = ["RANLIB=echo", "CDEFS=-DAdd_"]
+    # floating point opts
+    optf = ["OPTF=-O"]
+    gcc_major_ver = Formula["gcc"].any_installed_version.major
+    optf << "-fallow-argument-mismatch" if gcc_major_ver >= 10
```

Comment:
The current one is 13, so this doesn't have to be a conditional

Reply:
Forgive me, I'm not overly familiar with homebrew, but is there no scenario where a user could have an older GCC? Or is that a scenario that's just not covered?

Reply:
You're depending on gcc, so it'll never be lower than it is now (gcc 13)

Reply:
Makes sense, thanks!